### PR TITLE
Support TFJob/PytorchJob ActiveDeadlineSeconds

### PR DESCRIPTION
**Pull request by Syulin7**
tf-operator and pytorch-operator already support ActiveDeadlineSeconds.

1. support it in arena submit job.
2. pytorchjob default CleanPodPolicy change to Running

### CODE REVIEW

File: pkg/argsbuilder/submit_pytorchjob.go [hunk 1]
```diff
@@ -72,6 +72,7 @@ func (s *SubmitPytorchJobArgsBuilder) AddCommandFlags(command *cobra.Command) {
 	command.Flags().StringVar(&s.args.CleanPodPolicy, "clean-task-policy", "None", "How to clean tasks after Training is done, support None, Running, All.")
 	command.Flags().StringVar(&s.args.Cpu, "cpu", "", "the cpu resource to use for the training, like 1 for 1 core.")
 	command.Flags().StringVar(&s.args.Memory, "memory", "", "the memory resource to use for the training, like 1Gi.")
+	command.Flags().Int64Var(&s.args.ActiveDeadlineSeconds, "active-deadline-seconds", 0, "Specifies the duration (in seconds) since startTime during which the job can remain active before it is terminated")
```
Comment (cheyang) [inline pkg/argsbuilder/submit_pytorchjob.go:75]: I think  the default value 0 for active-deadline-seconds will mislead the user. How about not setting default value? If it's not setting,  There is no ActiveDeadlineSeconds.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/argsbuilder/submit_pytorchjob.go (hunk 1 above)
Comment (Syulin7) [reply pkg/argsbuilder/submit_pytorchjob.go]: Done, If it's not setting, There is no ActiveDeadlineSeconds in job's CR.

File: pkg/argsbuilder/submit_tfjob.go [hunk 2]
```diff
@@ -130,6 +133,8 @@ func (s *SubmitTFJobArgsBuilder) AddCommandFlags(command *cobra.Command) {
 	command.Flags().MarkDeprecated("cleanTaskPolicy", "please use --clean-task-policy instead")
 	command.Flags().StringVar(&s.args.CleanPodPolicy, "clean-task-policy", "Running", "How to clean tasks after Training is done, support Running, None and All.")
 
+	command.Flags().DurationVar(&activeDeadline, "active-deadline", activeDeadline, "Specifies the duration since startTime during which the job can remain active before it is terminated(e.g. '5s', '1m', '2h22m').")
```
Comment (cheyang) [inline pkg/argsbuilder/submit_tfjob.go:136]: I suggest using the name `running-timeout` instead of `active-deadline`.

Comment (Syulin7) [reply pkg/argsbuilder/submit_tfjob.go]: Done

**Pull request merged**